### PR TITLE
Update de_DE.lang

### PR DESCRIPTION
**Pull request by tistGamer**
This is the new de_DE.lang file.

Changelog (Compressed):
- added lines the previous one didn't have
- all lines in this file are 100% translated (But there are about 5 lines missing which i couldn't find in the english file)

Info: 
- This is file might get some updates
- All reoprted issues to my version of this file should go to me (if there is no problem)

--------------------------------------------------------------------------------------------------------------------------

Thats it!

### CODE REVIEW

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang [hunk 1]
```diff
@@ -1,30 +1,31 @@
-#en_US Language File for Ars Magica 2
+#de_DE Language File for Ars Magica 2
 
 #items
 item.arsmagica2:ArcaneCompendium.name=Arkanes Kompendium
-item.arsmagica2:FlickerFocusPrefix=Flickerfokus: %s
-item.arsmagica2:FlickerOperatorButchery.name=Schlachten
+item.arsmagica2:FlickerFocusPrefix=Flimmernder Fokus: %s
```
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: I lik the word "Flickerfokus", it sounds neat...

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang [hunk 2]
```diff
@@ -103,98 +108,102 @@ item.arsmagica2:mana_potion_standard.name=Standardmanatrank
...
-item.arsmagica2:sigil_felledoak=der gefällten Eiche
-item.arsmagica2:sigil_flatlands=des Flachen Landes
-item.arsmagica2:sigil_gentlerains=des Freundlichen Regens
-item.arsmagica2:sigil_interdiction=des Verbots
-item.arsmagica2:sigil_light=des Lichts
-item.arsmagica2:sigil_moonstoneattraction=der Mondgezeiten
-item.arsmagica2:sigil_naturesbounty=der Fülle der Natur
-item.arsmagica2:sigil_packedearth=der gepackted Erde
-item.arsmagica2:sigil_progeny=der Nachkommenschaft
+item.arsmagica2:sigil_base=Sigil der %s
```
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: "Sigil" is not a German word I know of. Should still be "Siegel"

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang [hunk 3]
```diff
@@ -103,98 +108,102 @@ item.arsmagica2:mana_potion_standard.name=Standardmanatrank
...
-item.arsmagica2:sunstone.name=Sonnstein
+item.arsmagica2:sunstone.name=Sonnenstein
 item.arsmagica2:unknown.name=Unbekannt
-item.arsmagica2:unknownEssence.name=Unbekannte Essenz
 item.arsmagica2:vinteumDust.name=Vinteumstaub
 item.arsmagica2:wakebloom.name=Wachblüte
 item.arsmagica2:warding_candle.name=Bannkerze
 item.arsmagica2:waterEssence.name=Wasseressenz
-item.arsmagica2:water_orbs.name=§6Wasserorb§f
+item.arsmagica2:water_orbs.name=§6Wasser Orbs§f
```
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: This is just wrong. It should stay the way it was before.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang [hunk 4]
```diff
@@ -103,98 +108,102 @@ item.arsmagica2:mana_potion_standard.name=Standardmanatrank
...
 item.arsmagica2:whiteRune.name=Weiße Rune
 item.arsmagica2:winter_arm.name=§6Griff des Winters§f
-item.arsmagica2:wizardChalk.name=Zauberkreide
+item.arsmagica2:wizardChalk.name=Zauberer's Kreide
```
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: That's not how you use Apostrophes in German. Please revert this change.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang [hunk 5]
```diff
@@ -103,98 +108,102 @@ item.arsmagica2:mana_potion_standard.name=Standardmanatrank
...
+tile.arsmagica2:cerublossom.name=Cerublossom
 tile.arsmagica2:crystalMarker.cm_export.name=Exportkristall
+tile.arsmagica2:crystalMarker.cm_final.name=Endgültiger Exportkristall
 tile.arsmagica2:crystalMarker.cm_import.name=Importkristall
-tile.arsmagica2:crystalMarker.cm_likeExport.name=Imitations-Exportkristall
-tile.arsmagica2:crystalMarker.cm_regulateExport.name=Regulierender Exportkristall
-tile.arsmagica2:crystalMarker.cm_regulateMulti.name=Bidirektional regulierender Kristall
-tile.arsmagica2:crystalMarker.cm_setExport.name=Einstellbarer Exportkristall
-tile.arsmagica2:crystalMarker.cm_setImport.name=Einstellbarer Importkristall
+tile.arsmagica2:crystalMarker.cm_likeExport.name=Exportkristall Imitat
```
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: This translation is misleading, it implies the "Imitat" is referring to the crystal itself, not the thing it targets.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang [hunk 6]
```diff
@@ -103,98 +108,102 @@ item.arsmagica2:mana_potion_standard.name=Standardmanatrank
...
+tile.arsmagica2:keystoneTrapdoor.name=Schlussstein Falltür
+tile.arsmagica2:keystonechest.name=Schlusssteintruhe
 tile.arsmagica2:lectern.name=Pult
-tile.arsmagica2:liquidEssence.name=Flüssiges Ätherium
+tile.arsmagica2:liquidEssence.name=Flüssiges Etherium
+tile.arsmagica2:magetorch.name=Magier-Licht
 tile.arsmagica2:magicWall.name=Magische Wand
-tile.arsmagica2:magic_sigil.name=Siegelstand
-tile.arsmagica2:magiciansWorkbench.name=Magierwerkbank
+tile.arsmagica2:magic_sigil.name=Sigil Stand
```
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: This is not even translated.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang [hunk 7]
```diff
@@ -103,98 +108,102 @@ item.arsmagica2:mana_potion_standard.name=Standardmanatrank
...
+tile.arsmagica2:magiciansWorkbench.name=Zauberkünstler's Werkbank
```
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: And please no apostrophe here, either.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang [hunk 8]
```diff
@@ -223,30 +234,31 @@ tile.arsmagica2:wakebloom.name=Wachblüte
 tile.arsmagica2:witchwoodSingleSlab.name=Hexholzstufe
 tile.arsmagica2:witchwoodleaves.name=Hexbaumblätter
 tile.arsmagica2:witchwoodlog.name=Hexbaumstamm
-tile.arsmagica2:wizardchalk.name=Zauberkreiderune
+tile.arsmagica2:wizardChalk.name=Zauberer's-Kreide Rune
 
 #fluids
-fluid.liquidEssence=Flüssiges Ätherium
+fluid.liquidEssence=Flüssiges Etherium
```
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: "Ether" in German would be "Äther", considering this word's origin.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang [hunk 9]
```diff
@@ -223,30 +234,31 @@ tile.arsmagica2:wakebloom.name=Wachblüte
...
-entity.arsmagica2.BossLightningGuardian.name=Donnerwächter
+entity.arsmagica2.BossLightningGuardian.name=Blitzwächter
 entity.arsmagica2.BossNatureGuardian.name=Naturwächter
 entity.arsmagica2.BossTowerGuardian.name=Turmwächter
 entity.arsmagica2.BossWaterGuardian.name=Wasserwächter
 entity.arsmagica2.BossWinterGuardian.name=Winterwächter
+entity.arsmagica2.DaBroom.name=Magischer Besen
 entity.arsmagica2.EarthElemental.name=Erdelementar
-entity.arsmagica2.Flicker.name=Flicker
+entity.arsmagica2.Flicker.name=Flimmern
```
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Again, I did like the word "Flickern"....

**Comment by Vexatos** [comment]
Apart from the things I left notes at, this looks rather well-done to me. Please consider fixing up the translation (don't forget to re-do the words you change everywhere in the translation, not just where I noted it).

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 1 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 29.10.2015 um 19:30 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43426054>:
>
> >
> >  #items
> >  item.arsmagica2:ArcaneCompendium.name=Arkanes Kompendium
> > -item.arsmagica2:FlickerFocusPrefix=Flickerfokus: %s
> > -item.arsmagica2:FlickerOperatorButchery.name=Schlachten
> > +item.arsmagica2:FlickerFocusPrefix=Flimmernder Fokus: %s
>
> I lik the word "Flickerfokus", it sounds neat...
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43426054>.
>
Flickerfokus oder Flimmernder Fokus... Das ist Ansichtssache, außerdem 
wäre das die Übersetzung.(Flimmernder Fokus)

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 3 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 29.10.2015 um 19:32 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43426260>:
>
> >  item.arsmagica2:vinteumDust.name=Vinteumstaub
> >  item.arsmagica2:wakebloom.name=Wachblüte
> >  item.arsmagica2:warding_candle.name=Bannkerze
> >  item.arsmagica2:waterEssence.name=Wasseressenz
> > -item.arsmagica2:water_orbs.name=§6Wasserorb§f
> > +item.arsmagica2:water_orbs.name=§6Wasser Orbs§f
>
> This is just wrong. It should stay the way it was before.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43426260>.
>
Werde ich machen...Wenn ich nur wüsste, warum ich das hiebei gemacht habe.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 4 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 29.10.2015 um 19:32 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43426298>:
>
> >  item.arsmagica2:whiteRune.name=Weiße Rune
> >  item.arsmagica2:winter_arm.name=§6Griff des Winters§f
> > -item.arsmagica2:wizardChalk.name=Zauberkreide
> > +item.arsmagica2:wizardChalk.name=Zauberer's Kreide
>
> That's not how you use Apostrophes in German. Please revert this change.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43426298>.
>
Mein Fehler...

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 2 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 29.10.2015 um 19:31 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43426180>:
>
> >  item.arsmagica2:redRune.name=Rote Rune
> >  item.arsmagica2:runebag.name=Runenbeutel
> > -item.arsmagica2:sigil_base=Siegel %s
> > -item.arsmagica2:sigil_butchery=des Abschlachtens
> > -item.arsmagica2:sigil_containment=des Behaltens
> > -item.arsmagica2:sigil_felledoak=der gefällten Eiche
> > -item.arsmagica2:sigil_flatlands=des Flachen Landes
> > -item.arsmagica2:sigil_gentlerains=des Freundlichen Regens
> > -item.arsmagica2:sigil_interdiction=des Verbots
> > -item.arsmagica2:sigil_light=des Lichts
> > -item.arsmagica2:sigil_moonstoneattraction=der Mondgezeiten
> > -item.arsmagica2:sigil_naturesbounty=der Fülle der Natur
> > -item.arsmagica2:sigil_packedearth=der gepackted Erde
> > -item.arsmagica2:sigil_progeny=der Nachkommenschaft
> > +item.arsmagica2:sigil_base=Sigil der %s
>
> "Sigil" is not a German word I know of. Should still be "Siegel"
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43426180>.
>
Ich habe das Wort bisher nicht gehört, geschweige denn in einem 
Wörterbuch gefunden. Allerdings hätte ich mich an 
"item.arsmagica2:sigil_base" orientieren können.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 8 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 29.10.2015 um 19:35 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43426683>:
>
> >
> >  #fluids
> > -fluid.liquidEssence=Flüssiges Ätherium
> > +fluid.liquidEssence=Flüssiges Etherium
>
> "Ether" in German would be "Äther", considering this word's origin.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43426683>.
>
Man kann sowohl "Ether", als auch "Äther" schreiben. Ich habe mich dann 
für Ether entschieden, weil mir die Schreibweise besser gefällt.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 9 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 29.10.2015 um 19:36 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43426718>:
>
> >  entity.arsmagica2.EarthElemental.name=Erdelementar
> > -entity.arsmagica2.Flicker.name=Flicker
> > +entity.arsmagica2.Flicker.name=Flimmern
>
> Again, I did like the word "Flickern"....
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43426718>.
>
Ich muss nichts zu eigener Meinung sagen, oder?

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 7 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 29.10.2015 um 19:34 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43426558>:
>
> >  tile.arsmagica2:magicWall.name=Magische Wand
> > -tile.arsmagica2:magic_sigil.name=Siegelstand
> > -tile.arsmagica2:magiciansWorkbench.name=Magierwerkbank
> > +tile.arsmagica2:magic_sigil.name=Sigil Stand
> > +tile.arsmagica2:magiciansWorkbench.name=Zauberkünstler's Werkbank
>
> And please no apostrophe here, either.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43426558>.
>
Auch hier, mein Fehler.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 6 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 29.10.2015 um 19:34 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43426538>:
>
> >  tile.arsmagica2:magicWall.name=Magische Wand
> > -tile.arsmagica2:magic_sigil.name=Siegelstand
> > -tile.arsmagica2:magiciansWorkbench.name=Magierwerkbank
> > +tile.arsmagica2:magic_sigil.name=Sigil Stand
>
> This is not even translated.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43426538>.
>
Wenn ich genauer darüber nachdenke, liegt es hier wieder am Wörtchen 
"Sigil". Das Wort habe ich noch nie gehört,
es in einem Wörterbuch gefunden, noch es mit dem Wort "Siegel" in 
Verbindung gebracht, deswegen habe ich gedacht, dass es sich hierbei um 
einen Fehler Ihrer Seits handelt.

Ich denke aber, ich kann mit Siegel aber gut leben, deshalb werde ich es 
beim nächsten Update der Datei ändern.
Vielen Dank für Ihre Hilfe, allerdings sind die Wörter "Flicker" und 
"Äther" Ansichtssache
und deshalb fänd ich es angebracht, mich nicht auf sowas hinzuweisen.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 5 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 29.10.2015 um 19:34 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43426477>:
>
> >  tile.arsmagica2:crystalMarker.cm_import.name=Importkristall
> > -tile.arsmagica2:crystalMarker.cm_likeExport.name=Imitations-Exportkristall
> > -tile.arsmagica2:crystalMarker.cm_regulateExport.name=Regulierender Exportkristall
> > -tile.arsmagica2:crystalMarker.cm_regulateMulti.name=Bidirektional regulierender Kristall
> > -tile.arsmagica2:crystalMarker.cm_setExport.name=Einstellbarer Exportkristall
> > -tile.arsmagica2:crystalMarker.cm_setImport.name=Einstellbarer Importkristall
> > +tile.arsmagica2:crystalMarker.cm_likeExport.name=Exportkristall Imitat
>
> This translation is misleading, it implies the "Imitat" is referring 
> to the crystal itself, not the thing it targets.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43426477>.
>
Ja, ich merke, dass es hier keinen Sinn macht, Imitat zu schreiben. Wäre 
dies der Fall, müsste es sich dabei um eine Art unechten Exportkristall 
handeln.

----------------------------------------------------------

Ich fasse jetzt mal alle 9 Nachrichten zusammen:

3/9 sind Geschmackssache(somit kein Fehler) -> Äther /*Wobei ich das 
eventuell ändern werde*/ u. Flicker
2/9 sind eine Grauzone, allerdings finde ich es gut sich darüber 
auszutauschen -> Sigil
1/9 ist ein (sagen wir mal) dummer Fehler ->  Imitat //Hierbei denke ich 
mir, warum ich solche Fehler mache.
3/9 sind wirkliche Fehler zu denen ich stehe und sie verbessern möchte 
-> Apostroph u. Wasserorb

Ich bitte demnächst nur um (dumme..., aber trotzdem) Fehler und 
eventuelle Grauzonen. Ansichtssachen sind vom menschen abhängig. Ich 
hoffe ich konnte/kann helfen.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 1 above)
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Sure, but there is no need to translate everything word-by-word.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 2 above)
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: "Sigil" in German would still be "Siegel", no matter what.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 5 above)
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Exactly.

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 6 above)
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: http://de.pons.com/%C3%BCbersetzung?q=sigil&l=deen&in=&lf=de

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 8 above)
Comment (Vexatos) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: But "Ether" is the name of a chemical, while "Äther" actually has a closer resemblance to the "ethereal".

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 6 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 30.10.2015 um 07:04 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43474054>:
>
> >  tile.arsmagica2:magicWall.name=Magische Wand
> > -tile.arsmagica2:magic_sigil.name=Siegelstand
> > -tile.arsmagica2:magiciansWorkbench.name=Magierwerkbank
> > +tile.arsmagica2:magic_sigil.name=Sigil Stand
>
> http://de.pons.com/%C3%BCbersetzung?q=sigil&l=deen&in=&lf=de
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43474054>.
>
Ok. Eigentlich habe ich dort genau nach diesem Wort gesucht, allerdings 
nicht gefunden. Danke!

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 8 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 30.10.2015 um 07:05 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43474117>:
>
> >
> >  #fluids
> > -fluid.liquidEssence=Flüssiges Ätherium
> > +fluid.liquidEssence=Flüssiges Etherium
>
> But "Ether" is the name of a chemical, while "Äther" actually has a 
> closer resemblance to the "ethereal".
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43474117>.
>
Wie ich schon in meiner letzten Nachricht geschrieben habe, werde ich 
das ändern.(Vielleicht haben Sie es einfach übersehen).

File: src/main/resources/assets/arsmagica2/lang/de_DE.lang (hunk 1 above)
Comment (tistGamer) [inline src/main/resources/assets/arsmagica2/lang/de_DE.lang]: Am 30.10.2015 um 07:02 schrieb Vexatos:
>
> In src/main/resources/assets/arsmagica2/lang/de_DE.lang 
> <https://github.com/Mithion/ArsMagica2/pull/1335#discussion_r43474015>:
>
> >
> >  #items
> >  item.arsmagica2:ArcaneCompendium.name=Arkanes Kompendium
> > -item.arsmagica2:FlickerFocusPrefix=Flickerfokus: %s
> > -item.arsmagica2:FlickerOperatorButchery.name=Schlachten
> > +item.arsmagica2:FlickerFocusPrefix=Flimmernder Fokus: %s
>
> Sure, but there is no need to translate everything word-by-word.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Mithion/ArsMagica2/pull/1335/files#r43474015>.
>
Mal sehen....
Aber irgendwie find ich, dass es sich nicht so schlimm anhört und es im 
Prinzip auch richtig übersetzt ist.
Ist aber erstmal zweitrangig.

**Comment by tistGamer** [comment]
Here it is: THE UPDATE IT NEEDS

It changed all misstakes.(All I've found and all you've shown me)
That's embarrassing...Well, but misstakes teach us to do them not again.

**Comment by tistGamer** [comment]
UPDATE: I changed my messages of this conversation to english.
